scss/titles.scss

```diff
@@ -2,6 +2,9 @@
 @import 'sizes';
 
 .title {
-	color: $color-title;
+	color: $black;
 	font-size: $size-title;
+	padding-left: 300px;
+	padding-top: 100px;
+	padding-bottom: 10px;
```

Comment:
Use the Padding shorthand Property: `padding: 100px ??px 10px 300px;`, changing `??` for the proper value.